pkg/auth/oauth/handlers/authenticator.go

```diff
@@ -23,6 +24,10 @@ func NewAuthorizeAuthenticator(request authenticator.Request, handler Authentica
 	return &AuthorizeAuthenticator{request, handler, errorHandler}
 }
 
+type ClientExpiration interface {
+	GetExpiration(int32) int32
```

Comment:
Is there any way to turn this into `GetExpiration() int32`?  Taking in the default seems meh.